modules/apps/portal-search/portal-search-rest-test/src/testIntegration/java/com/liferay/portal/search/rest/resource/v1_0/test/SearchResultResourceTest.java

```diff
@@ -1271,6 +1359,20 @@ null, null, _getSearchFacets(jsonObject),
 			jsonObject.getLong("totalCount"));
 	}
 
+	private static DefaultObjectEntryManager _defaultObjectEntryManager;
+	private static DTOConverterContext _dtoConverterContext;
```

Comment:
@JhossephG do we need this variables at class level? try to define this variables inside the test.

Thanks!